xml/ha_config_cli.xml

```diff
@@ -1378,11 +1378,11 @@ hostname (string): Hostname
     </para>
 <!-- order ID score-type: FIRST-RSC[:ACTION] THEN-RSC[:ACTION] 
      
-     score-type :: advisory | mandatory | <score>
+     score-type :: advisory | Mandatory | <score>
```

Comment:
Sensible to make this update,  although this seems to be an xml comment which is not really shown in the generated doc :-)

If to synchronize with crmsh, the current help text is like:

```
order <id> [kind:] first then [symmetrical=<bool>]
kind :: Mandatory | Optional | Serializ
```

Reply:
Thanks @gao-yan! Much appreciated! :+1: 

I'm wondering whether `Serializ` a typo? Should it be `Serialize`? :thinking:

Reply:
Oh, sorry. Failed copying/pasting  :-)

Of course, the complete text can be gotten with:
`crm configure help order`

Reply:
Great! Thanks! :+1: